time_tracker_api/__init__.py

```diff
@@ -1,12 +1,14 @@
 from flask import Flask
-from flask_restplus import Api
 
 
 def create_app():
     flask_app = Flask(__name__)
-    app = Api(app=flask_app)
 
-    from time_tracker_api.projects.projects_endpoints import ns as projects_ns
-    app.add_namespace(projects_ns)
+    init_app(flask_app)
 
     return flask_app
+
+
+def init_app(app):
```

Comment:
Could we use type hints here  https://docs.python.org/3/library/typing.html ?

Reply:
Makes sense, I will do it.